tools/psltool/psltool.go

```diff
@@ -154,7 +156,7 @@ func runValidate(env *command.Env, pathOrHash string) error {
 		bs, err = os.ReadFile(pathOrHash)
 	} else if isHex(pathOrHash) {
 		// input looks like a git hash
-		client := github.Client{}
+		client := github.GHRepo{}
```

Comment:
I think this client needs the owner/repo fields as well, otherwise PSLForHash will look up in the wrong repo